recipe/meta.yaml

```diff
@@ -24,13 +24,17 @@ requirements:
     - gcc  # [osx]
     - boost 1.63.*  # note this is intentionally NOT the latest version; synced with casacore which has special needs
     - casacore 2.2.*
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
```

Comment:
OK. I think I get the gist but from your description I'm still a bit confused about what requirements I could put instead, so I'll stick with the document ones for now.